boards/arm/stm32f072_eval/doc/stm32f072_eval.rst

```diff
@@ -0,0 +1,195 @@
+- Four color user LEDs and two LEDs as MCU low power alarm
+- Extension connector for daughter board or wrapping board
+- MCU voltage choice fixed 3.3 V or adjustable from 1.65 V to 3.6 V
+- USB full-speed connector
+- Touch sensing buttons
+- RTC with backup battery
+- CAN2.0A/B compliant connector
+- Light Dependent Resistor (LDR)
+- Potentiometer
+- Two HDMI connectors with DDC and CEC
+- Smart Card slot
+- Motor control connector
+
+
+.. image:: img/stm32f072_eval.jpg
+     :width: 540px
+     :align: center
+     :height: 549px
+     :alt: STM32F072-EVAL
+
+More information about the board can be found at the `STM32F072-EVAL website`_.
+
+Hardware
+********
+
+STM32F072-EVAL Discovery kit provides the following hardware components:
+
+- STM32F072VBT6 in LQFP100 package
+- ARM |reg| 32-bit Cortex |reg| -M0 CPU
+- 48 MHz max CPU frequency
+- VDD from 2.0 V to 3.6 V
+- 128 KB Flash
+- 16 KB SRAM with HW parity
+- GPIO with external interrupt capability
+- one 12-bit ADC with 16 channels
+- one 12-bit D/A converters with 2 channels
+- RTC
+- Advanced-control Timer
+- General Purpose Timers (8)
+- Watchdog Timers (2)
+- USART (4)
+- I2C (2)
+- SPI (2)
+- CAN
+- USB 2.0 OTG FS with on-chip PHY
+- CRC calculation unit
+- DMA Controller
+- HDMI CEC Controller
+- 24 capacitive sensing channels for touchkey, linear and rotary touch sensors
+- Up to 87 fast I/Os, 68 I/Os with 5V tolerant capability and 19 with independent supply
+
+More information about STM32F072VB can be found here:
+       - `STM32F072VB on www.st.com`_
+       - `STM32F072 reference manual`_
+
+Supported Features
+==================
+
+The Zephyr stm32f072_eval board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| PWM       | on-chip    | pwm                                 |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | reset and clock control             |
++-----------+------------+-------------------------------------+
+| IWDG      | on-chip    | independent watchdog                |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on Zephyr porting.
```

Comment:
change "on Zephyr porting" to "on this Zephyr porting"

Reply:
Done